### PR TITLE
Highlight fields in error and repeat error message under them

### PR DESCRIPTION
**Pull request by danmassie**

### CODE REVIEW

File: app/views/fields/checkbox.scala.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+* @param args Set of extra HTML attributes ('''id''' and '''label''' are 2 special arguments).
+* @param handler The field constructor.
+*@
+@(field: play.api.data.Field, args: (Symbol,Any)*)(implicit handler: FieldConstructor, messages: play.api.i18n.Messages)
+
+@boxValue = @{ args.toMap.get('value).getOrElse("true") }
+
+@input(field, args:_*) { (id, name, value, htmlArgs) =>
+    <input type="checkbox" id="@id" name="@name" value="@boxValue" @if(value == Some(boxValue)){checked="checked"} @toHtmlArgs(htmlArgs.filterKeys(_ != 'value))/>
+}
```
Comment (susiecoleman) [inline app/views/fields/checkbox.scala.html]: Is this used?

File: conf/messages.en [hunk 2]
```diff
@@ -52,8 +48,15 @@ editUser.userGroupsHeader=User Groups
...
-error.username=Username is invalid
+email=Email Address
+username=Username
+personalDetails.firstName=First name
+personalDetails.lastName=Last name
+status.receiveGnmMarketing=Receive GNM marketing
+status.receive3rdPartyMarketing=Receive 3rd party marketing
+
+error.email=A valid email address is required
+error.username=Username is required and must be alphanumeric and between 6 and 20 characters
```
Comment (susiecoleman) [inline conf/messages.en]: Are these written in this format so they can be automatically used by the @inputText in the view? (Not a criticism of the format I'm just curious)

File: app/views/fields/checkbox.scala.html (hunk 1 above)
Comment (danmassie) [inline app/views/fields/checkbox.scala.html]: yes, see @fields.checkbox in editUser.scala.html

Comment (susiecoleman) [inline app/views/fields/checkbox.scala.html]: Found it - sorry my bad

**Comment by susiecoleman** [comment]
:+1:

**Pull request merged**